lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -245,7 +246,9 @@ def self.upload_binary(app_id, binary_path)
         end
       end
 
-      # When the binary is uploaded response body includes the release id and so that is stored for future upload methods.
+      # <"Uploads the binary">
+      #
+      # <"Returns the id of the release"> <explain "Only happens on a successful release">
```

Comment:
Oops! Sorry my comment was confusing but I didn't mean to actually include the `<>` and the quotation marks :)

Writing it in paragraph form would be great, for example:

```
# Uploads the binary
# 
# Returns the id of the release only on a successful release
```

Reply:
I thought it was some ruby doc format, I will fix it now.